components/FullRolesCarousel.jsx

```diff
@@ -0,0 +1,62 @@
+import { SwiperSlide } from "swiper/react";
+import Tile from "../components/Tile";
+import Image from "next/image";
+import Carousel from "../components/Carousel";
+import { Navigation } from 'swiper';
+import Button from "../components/Button";
+import { roleDescription } from "../data/roleDescription";
+import "swiper/css/navigation";
+
+const ROLE_TILE_SIZE = 450;
+
+
+const RolesTile = ({role}) => {
+    return(
```

Comment:
It looks like [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) isn't working to properly format the code (space everything out, etc). Do you have the [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension installed? Could also be a reason why the Vercel builds are failing